.vib/oauth2-proxy/cypress/cypress/integration/oauth2-proxy_spec.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+it('redirects to the auth provider login', () => {
+  cy.visit('/', {failOnStatusCode:false})
+  cy.contains('button', 'Sign in');
+
+  cy.request({
```

Comment:
Thinking about future maintenance, I would add a small comment here explaining why we are restricted to only perform this check.

Reply:
Agree, added!